packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -201,8 +220,7 @@ class Transforms {
         }
 
         // Decrypt activity message if present
-        // For 'add' activities the objectDisplayName is a user id which does not need to be decrypted
-        if (activity.verb !== 'add' && activity.objectDisplayName && activity.encryptionKeyUrl) {
+        if (activity.objectDisplayName) {
```

Comment:
Add/Leave/etc activities will no longer make it this far, and neither will activities without an encryptionKeyUrl